src/grafana/prometheus/docker/prometheus-local/prometheus-local-file.yml

```diff
@@ -12,3 +12,19 @@ scrape_configs:
       - files:
           - /etc/prometheus/targets.json
         refresh_interval: 30s
+
```

Comment:
Why do we need `prometheus-local-file.yml`? It is unclear how it is used.

Reply:
Yes `prometheus-locl-file.yml` is required and is in fact the only reason to build the `springcloud/spring-cloud-dataflow-prometheus-local` image instead of using the `prom/prometheus:v2.12.0` directly. 
The `prometheus-local-file.yml` file pre-configures Prometheus with the target systems to monitor. Later being the old custom Service-Discovery mechanism or the RSocket Proxy. 
Perhaps we can remove the `job_name: 'scdf'` definition as we removed the our home-grown  service-discovery service.